plugins/woocommerce/src/Internal/Admin/WcPayWelcomePage.php

```diff
@@ -68,6 +68,11 @@ public function must_be_visible(): bool {
 			return false;
 		}
 
+		// Temporary solution until we improve the dismiss and cache logic.
+		if (false !== get_option( 'wcpay_welcome_page_incentives_dismissed', false )) {
```

Comment:
Nitpick: code formatting here (spacing - should be `if ( false !== get_option( 'wcpay_welcome_page_incentives_dismissed', false ) ) {`

Reply:
Thanks for catching it, I forgot about having the formatter disabled 🙂 _Addressed in 50995e0_